subt/docker/robotika/ros_proxy_node.cc

```diff
@@ -375,7 +377,7 @@ void Controller::Update(const ros::TimerEvent&)
 
       // Create a cmd_vel publisher to control a vehicle.
       this->originClient = this->n.serviceClient<subt_msgs::PoseFromArtifact>(
-          "/subt/pose_from_artifact_origin");
+          "/subt/pose_from_artifact_origin", true);
```

Comment:
what "true" means here?

Reply:
see the commits - it makes the connection persistent